sdk/appcenter-data/src/main/java/com/microsoft/appcenter/data/Data.java

```diff
@@ -195,7 +195,21 @@ public static void setTokenExchangeUrl(String tokenExchangeUrl) {
      * If the operation fails, the error can be checked by reading {@link Page#getError()} on the first page of the results: {@link PaginatedDocuments#getCurrentPage()}.
      */
     public static <T> AppCenterFuture<PaginatedDocuments<T>> list(Class<T> documentType, String partition) {
-        return getInstance().instanceList(documentType, partition);
+        return list(documentType, partition, new ReadOptions());
```

Comment:
I'd prefer to use `instanceList` here too since it's one less level of indirection

```suggestion
        return getInstance().instanceList(documentType, partition, new ReadOptions());
```